lib/lighthouse/benefits_education/service.rb

```diff
@@ -49,6 +49,8 @@ def inspect
     def get_gi_bill_status
       raw_response = begin
         config.get(@icn)
+      rescue Breakers::OutageException => e
+        raise e
```

Comment:
Allow the `OutageException` to bubble up, to be handled by the `ApplicationController`